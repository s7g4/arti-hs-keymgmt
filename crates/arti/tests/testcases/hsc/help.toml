bin.name = "arti"
args = "hsc --help"
status.code = 0
