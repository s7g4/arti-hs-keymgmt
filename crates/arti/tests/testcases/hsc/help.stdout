Run state management commands for an Arti hidden service client

Usage: arti hsc [OPTIONS] <COMMAND>

Commands:
  get-key  Prepare a service discovery key for connecting to a service running in restricted discovery mode. (Deprecated: use `arti hsc key get` instead)
  key      Key management subcommands
  help     Print this message or the help of the given subcommand(s)

Options:
  -c, --config <FILE>                 Specify which config file(s) to read. Defaults to [File("[..]"), Dir("[..]")]
  -o <KEY=VALUE>                      Override config file parameters, using TOML-like syntax.
  -l, --log-level <LEVEL>             Override the log level (usually one of 'trace', 'debug', 'info', 'warn', 'error').
      --disable-fs-permission-checks  Don't check permissions on the files we use.
  -h, --help                          Print help
